Libs/MRML/Core/vtkMRMLDiffusionTensorVolumeDisplayNode.cxx

```diff
@@ -226,6 +227,14 @@ void vtkMRMLDiffusionTensorVolumeDisplayNode::UpdateImageDataPipeline()
 
 }
 
+//----------------------------------------------------------------------------
+void
+vtkMRMLDiffusionTensorVolumeDisplayNode::SetTensorRotationMatrix(vtkMatrix4x4 *rotationMatrix)
```

Comment:
It would be important to include the coordinate system and transform direction in the name: RAS/LPS (this is especially critical in the storage node) and ToRAS or FromRAS. For example, TensorMeasurementFrameToRASRotation or TensorToRASRotation names would be specific enough.

Reply:
Yes, we should rationalize all the naming conventions for these methods.  Some places use Measurement Frame, others use TensorRotationMatrix.  The wording I chose here seems most consistent with the rest of this API, which assumes all of this is in RAS.  Let's let @ljod look at this when she's back from vacation.

Reply:
If the same name is already used at many places and it is used fully consistently then it is probably not worth the trouble of changing the name now, just provide a clear definition in the API documentation.

Reply:
All the tensors and gradient calculations are in RAS and that's not explicit in any of the method names so this method name is consistent with the rest of the code.

Node documentation already specifies this:

https://github.com/Slicer/Slicer/blob/40087afee09fa4779e1b809423d927add1904cb7/Libs/MRML/Core/vtkMRMLDiffusionTensorVolumeNode.h#L29-L30